src/core.c

```diff
@@ -420,7 +414,7 @@ typedef struct CoreData {
 #if defined(PLATFORM_RPI) || defined(PLATFORM_DRM)
             int defaultMode;                // Default keyboard mode
             struct termios defaultSettings; // Default keyboard settings
-            KeyEventFifo lastKeyPressed;    // Buffer for holding keydown events as they arrive (Needed due to multitreading of event workers)
+			int fd;            				// File descriptor for the evdev keyboard
```

Comment:
Please, could you review TABS by 4 spaces?

Reply:
Oops - thought I'd got those. Fixing now.